tests/common.py

```diff
@@ -1,11 +1,12 @@
 import hashlib
+import os
 
 from data_diff.database import CHECKSUM_HEXDIGITS, MD5_HEXDIGITS
 import logging
 
 logging.basicConfig(level=logging.WARN)
 
-TEST_MYSQL_CONN_STRING = "mysql://mysql:Password1@localhost/mysql"
+TEST_CONN_STRING = os.environ.get("TEST_CONN_STRING", "mysql://mysql:Password1@localhost/mysql")
```

Comment:
It's probably better to remove the default and throw an error message Because the error "bad username" or "no such database" can be confusing, and no one will know to look here for it.

Reply:
Sounds reasonable. Will change that.

Reply:
My idea initially to use this as a default is that nothing would change. FYI ;)